client-js/src/error.ts

```diff
@@ -25,18 +25,20 @@ export enum ERROR_CODES {
   InvalidAdminList = 60011,
   /// The list of accounts that can mint tokens is invalid.
   InvalidMinterList = 60012,
-  ///  The list of accounts that can burn tokens is invalid.
-  InvalidBurnerList = 60013,
-  /// The list of accounts that can mint and burn is invalid.
-  InvalidMintAndBurnList = 60014,
   /// The list of accounts with no access rights is invalid.
-  InvalidNoneList = 60015,
+  InvalidNoneList = 60013,
```

Comment:
I am not sure Karan would allow to change errors identification here for backward compat ? but you know better then me how retro compat works, I guess not that important on installation but if this is used in security update I guess some people might use this identification.

Reply:
The errors stayed the same in the contract. It seems like there was discrepancy in the js client.